tests/nnfw_api/src/GenModelTests/one_op_trains/FullyConnected.test.cc

```diff
@@ -33,9 +33,12 @@ TEST_F(GenModelTrain, OneOp_FullyConnected)
 
   float learning_rate = 0.01f;
   int32_t batch_size = 1;
-  cgen.addTrainInfo({circle::Optimizer::Optimizer_SGD, learning_rate,
+  cgen.addTrainInfo({circle::Optimizer::Optimizer_SGD,
+                     learning_rate,
                      circle::LossFn::LossFn_MEAN_SQUARED_ERROR,
-                     circle::LossReductionType::LossReductionType_SumOverBatchSize, batch_size});
+                     circle::LossReductionType::LossReductionType_SumOverBatchSize,
+                     batch_size,
+                     {0}});
```

Comment:
(Optional)
How about adding a comment about what is {0}?
or 
adding the following line:
```c++
const auto training_ops = std::vector<int32_t>{0};
```

Reply:
not applied now, code refactored ;)